lib/src/config/config.rs

```diff
@@ -99,7 +99,7 @@ impl Config {
     /// returned:
     ///
     ///   * **address**: String
-    ///   * **port**: Integer
+    ///   * **port**: Unsigned 16bit Integer
```

Comment:
This should stay as `Integer` as it's referring to the `Value::Integer` variant.